config.json

```diff
@@ -180,6 +180,23 @@
                 "MAX_LIMIT": 2,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "FERDYNET",
+            "NAME": "FerdyFlip Testnet",
+            "TOKEN": "FAVAX",
+            "RPC": "https://testnet-rpc.ferdyflip.xyz/rpc",
+            "CHAINID": 668577,
+            "EXPLORER": "https://devnet.routescan.io",
```

Comment:
Can we have actual explorer link here, so that users can view transaction details for their faucet drip?

Reply:
Cool. Np, it's just that users won't be able to see drip receipt, otherwise shouldn't be a blocker 👍

Reply:
Thanks! Will send a follow up when available.